overrides/build-systems.json

```diff
@@ -16265,7 +16265,8 @@
   "propcache": [
     "cython",
     "expandvars",
-    "setuptools"
+    "setuptools",
+    "tomli"
```

Comment:
Since this is Python version specific it should go in `overrides/default.nix` and be selectively added to `nativeBuildInputs` using `lib.optionals`.

Reply:
Fixed